ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,78 +1,80 @@
-===================================================
-How to manage orders paid with payment acquirers
-===================================================
+=========================================
+Manage orders paid with Payment Acquirers
+=========================================
 
-Odoo confirms orders automatically as soon as the payment is authorized 
+Odoo automatically confirms orders as soon as the payment is authorized 
 by a payment acquirer. This triggers the delivery.
-If you invoice based on ordered quantities,
-you are also requested to invoice the order.
+If you invoice based on ordered quantities, you are also requested to invoice the order.
 
+Checking the status of a payment
+================================
+In `Debug <https://odoo.com/documentation/user/14.0/general/developer_mode/activate.html>`_
+mode, the user can check the transaction status from the order, by selecting
+*Payment Transactions* under the *Configuration* menu.
```

Comment:
Developer/debug mode is not the normal flow. If it's hidden from users, it's probably for a reason. There may be additional views hidden from most users but that doesn't mean it's the normal flow. In my understanding of SO, the payment is confirmed with an automatic note in the chatter, and if you create an invoice, the payment is directly reconciled.

I guess - not sure about it - that in older versions of odoo, the transaction was directly shown on the SO as the current screenshot suggests. These states were probably visible then.

Reply:
Ah, I see what you mean. Makes sense. So, seeing as how this is only accessible through Debug mode, and you mentioned that this first point was no longer relevant anyway, I will go ahead and remove this portion of the documentation. But, I will mention how the payment is confirmed in the Chatter of an SO (via an automatic note). And, I will also mention that if they create an invoice, a payment will be directly reconciled immediately.